cpp-terminal/terminal.h

```diff
@@ -352,7 +351,6 @@ class Terminal: public BaseTerminal {
             case '\x09': // TAB
                 return Key::TAB;
             case '\x0a': // LF
```

Comment:
If you remove the `return`, I think it's a good habit to document that the `case` is meant to fall through:
```suggestion
            case '\x0a': // LF; falls-through
```